rpyscope/add_widgets.py

```diff
@@ -45,7 +45,7 @@ def browse_history(self, up=True):
 
     def keyPressEvent(self, a0):
         """Handle key press events -> go to _history for arrow up and down."""
-        super(LineEditHistory, self).keyPressEvent(a0)
+        super().keyPressEvent(a0)
```

Comment:
I didn't read the command window code so I will just trust you there.

Reply:
These super calls don't need these arguments, it is a style change introduced by by requiring at least python 3.6 to be installed (which should be default by now, since both py36 and py37 are end of life). See [here](https://github.com/asottile/pyupgrade)